shipyard/src/main/java/org/jclouds/shipyard/handlers/ShipyardErrorHandler.java

```diff
@@ -69,6 +70,12 @@ public void handleError(HttpCommand command, HttpResponse response) {
                if (command.getCurrentRequest().getMethod().equals("POST")) {
                   if (command.getCurrentRequest().getEndpoint().getPath().endsWith("/engines")) {
                      exception = new HttpResponseException("Connection refused registering docker daemon", command, response);
+                  } else if (command.getCurrentRequest().getEndpoint().getPath().endsWith("/containers")) {
+                     exception = new HttpResponseException(command, response, message);
+                  }
+               } else if (command.getCurrentRequest().getMethod().equals("DELETE")) {
+                  if (command.getCurrentRequest().getEndpoint().getPath().endsWith("/servicekeys")) {
+                     exception = new ResourceNotFoundException(message, new HttpResponseException(command, response, message));
```

Comment:
Is this `ResourceNotFoundException` the only possible error here, or should we inspect the response message to know when to throw the RNFE or the default one?